scripts/src/public/services/orders.ts

```diff
@@ -103,6 +106,13 @@ export async function changeOrder(orderId: string, userId: string, change: Parti
 }
 
 async function createOrder(appOffer: AppOffer, user: User, orderTranslations = {} as OrderTranslations) {
+	const app = (await Application.findOneById(user.appId))!;
+	if (appOffer.offer.type === "earn") {
+		throwOnAppEarnLimit(app.id, "total_earn", app.config.limits.minute_total_earn, moment.duration({ minutes: 1 }), appOffer.offer.amount);
+		throwOnAppEarnLimit(app.id, "total_earn", app.config.limits.hourly_total_earn, moment.duration({ hours: 1 }), appOffer.offer.amount);
+		throwOnUserEarnLimit(user.id, "user_earn", app.config.limits.hourly_user_earn, moment.duration({ hours: 1 }), appOffer.offer.amount);
```

Comment:
didn't we want this on a daily limit of 500?

Reply:
To change it to daily?